wasp/apps/clock.py

```diff
@@ -21,7 +21,7 @@
         digits.clock_8, digits.clock_9
 )
 
-MONTH = 'JanFebMarAprMayJunJulAugSepOctNovDec'
```

Comment:
" for ' ... yes indeed habits. Used to use " for natural text. And because that s becoming a standard in Python community.

This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

Reply:
I think this entire file change can be dropped, as it's only formatting